sdk/eventgrid/azure-eventgrid/samples/consume_cloud_events_from_service_bus_queue.py

```diff
@@ -0,0 +1,43 @@
+import sys
+import os
+import datetime as dt
+import json
+
+PACKAGE_PARENT = '..'
+SCRIPT_DIR = os.path.dirname(os.path.realpath(os.path.join(os.getcwd(), os.path.expanduser(__file__))))
+sys.path.append(os.path.normpath(os.path.join(SCRIPT_DIR, PACKAGE_PARENT)))
+
+from msrestazure.azure_active_directory import ServicePrincipalCredentials
+from azure.core.pipeline.policies import AzureKeyCredentialPolicy
+from azure.core.credentials import AzureKeyCredential
+from azure.core.exceptions import (
+    ResourceNotFoundError,
+    ResourceExistsError,
+    ClientAuthenticationError
+)
+
+from azure.eventgrid._consumer import EventGridConsumer
+from azure.servicebus import ServiceBusClient, Message
+
+def get_deserialized_events(msg):
+    events = consumer.deserialize_events(msg)
+    for event in events:
+        if 'specversion' in event:
```

Comment:
That check should be part of the sample, we probably want to re-think our pedagogy to the customer